src/latexify/codegen/function_codegen_test.py

```diff
@@ -862,6 +862,34 @@ def test_use_set_symbols_compare(code: str, latex: str) -> None:
     assert function_codegen.FunctionCodegen(use_set_symbols=True).visit(tree) == latex
 
 
+def test_generate_numpy_zeros():
```

Comment:
Please write comprehensive tests to cover every edge cases, as in other parametric tests.